common/lib/driver-definitions/package.json

```diff
@@ -86,10 +86,6 @@
           "forwardCompat": false,
           "backCompat": false
         },
-        "InterfaceDeclaration_IDocumentStorageService": {
```

Comment:
looks like we actually had an invalid suppression

Reply:
🤔